src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -28,6 +28,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string ImplementInterface = "Implement Interface";
         public const string InsertMissingCast = "InsertMissingCast";
         public const string MoveToTopOfFile = "Move To Top Of File";
+        public const string QualifyMemberAccess = "Qualify member access";
```

Comment:
I was under the impression that these strings needed to be human-readable because they're set as the name of the code fix provider in [`ExportCodeFixProviderAttribute`](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeFixes/ExportCodeFixProviderAttribute.cs,49010341106c0e4e).